### PR TITLE
Feature/calendar

### LINKED ISSUE #46: As an initiator I want to get an update of calendar availability so that I can see the availability of all parties.

### PR DESCRIPTION
**Pull request by thejinxters**
closes #46

### CODE REVIEW

**Comment by thejinxters** [comment]
Right now it is just sending back dummy data, but there are other stories to properly gather and find the best time. I figure this can be done for right now.

File: server/helpers/auth/google.coffee [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    if callback
+      return callback err, tokens
+
+buildAuthClient = (clientId, redirectUri)->
+  secret = config.googleAuthConfig.clientSecret
+  OAuth2 = googleapis.auth.OAuth2
+  oauth2Client = new OAuth2 clientId, secret, redirectUri
+  return oauth2Client
+
+getUsersCalendars = (userList, eventsList, currentUser, callback) ->
```
Comment (Zandrr) [inline server/helpers/auth/google.coffee:80]: Do we wanna do a promise here?

Comment (thejinxters) [inline server/helpers/auth/google.coffee:80]: Yes, I want to refactor everything so it's not a bunch of nested callbacks

Comment (Zandrr) [inline server/helpers/auth/google.coffee:80]: For now, it looks good to me. lol @ your signature for github comments.

Comment (Zandrr) [inline server/helpers/auth/google.coffee:80]: Oh it must be you responding via email!

**Comment by Zandrr** [comment]
SHIP

**Pull request merged**